### ISSUE TITLE
Water calibration - update parameter file IBL-wide

### ISSUE THREAD

**Issue by GaelleChapuis** [int-brain-lab/iblrig#96]
10 ms is too short, it should have been changed into longer opening times. Start with 50 and go up in steps of 50 up until 200 ms.

It seems important that the changes to the calibration file is made IBL-wide. 
Please update calibration param file and release new version.

```
MIN_OPEN_TIME = 50  # (ms)
MAX_OPEN_TIME = 201  # (ms)
STEP = 50  # (ms)
PASSES = 3  # number of times to repeat the same open_time per run
```

**Comment by anne-urai** [comment]
https://github.com/int-brain-lab/iblrig/issues/89

**Comment by nbonacchi** [comment]
fixed already in 3.5.3
